src/app/api/_api-services/auth_service/index.ts

```
@@ -379,6 +382,88 @@ export class AuthService {
+					...user,
+					loginAddress: formattedAddress,
+					loginWallet: wallet
+				})
+			};
+		}
+
+		// user does not exist, register
+		const username = `${formattedAddress.substring(0, 6)}...${formattedAddress.substring(formattedAddress.length - 4)}`; // example: 5Grwva...KpZf3 or 0x0000...0001
+		const password = createCuid();
+
+		const newUser = await this.CreateUser({
```

Biome (1.9.4)

[error] Using this in a static context can be confusing.

this refers to the class.
Unsafe fix: Use the class name instead.


(lint/complexity/noThisInStatic)